### PR TITLE
Umcs 451/upl invoice type

### PR DESCRIPTION
**Pull request by Ryouzanpaku**

### CODE REVIEW

File: test/integration/PaymentTypes/InvoiceGuaranteedTest.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+class InvoiceGuaranteedTest extends BaseIntegrationTest
+{
+    /**
+     * Verify, backwards compatibility regarding fetching payment type and map it to invoice secured class.
+     *
+     * @test
+     */
+    public function ivgTypeShouldBeFechable(): InvoiceSecured
+    {
+        $ivgMock = $this->getMockBuilder(InvoiceSecured::class)->setMethods(['getUri'])->getMock();
```
Comment (sixer1182) [inline test/integration/PaymentTypes/InvoiceGuaranteedTest.php:43]: Add to changelog

**Review by sixer1182 — CHANGES_REQUESTED** [review]
Very good 💯 
Just missing the changlog entry for ivg test extraction.

File: test/integration/PaymentTypes/InvoiceGuaranteedTest.php (hunk 1 above)
Comment (Ryouzanpaku) [reply test/integration/PaymentTypes/InvoiceGuaranteedTest.php:43]: I dont realy saw this noetworthy in the changelog. I would prefere to let it go under "several minior improvements".

Comment (Ryouzanpaku) [reply test/integration/PaymentTypes/InvoiceGuaranteedTest.php:43]: Will be added with the next part of the UPL integration

**Review by sixer1182 — APPROVED** [review]

**Pull request merged**